pagopa/io-app
# chore: [IOBP-493,IOBP-491] Cleanup after BPD and BV removal

## Short description
thorough cleanup after the removal of BPD and BV,
DEPENDS ON https://github.com/pagopa/io-app/pull/5432

## List of changes proposed in this pull request
- removed scripts
- updated config files
- updated env files
- removed i18n entries
- removed mocks where possible
- removed now completely useless client and actions
- removed now useless saga

## How to test
as usual, run the tests and have a look around the app, nothing should break or change from an user's perspective
